k8s.old/scalyr-agent-2.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: extensions/v1beta1
```

Comment:
`k8s.old` is too generic... the definition of `old` will change over time.

What's the definition of old here?  Whether or not they used the old agent setup instructions?  If so, then we should maybe call it something like `pre-2.1` instructions or something like that.